modules/ts/src/ts_gtest.cpp

```diff
@@ -4054,7 +4054,7 @@ enum GTestColor {
   COLOR_YELLOW
 };
 
-#if GTEST_OS_WINDOWS && !GTEST_OS_WINDOWS_MOBILE
+#if GTEST_OS_WINDOWS && !GTEST_OS_WINDOWS_MOBILE && !defined WINRT
```

Comment:
This should work however this may be confusing as these are two different OS. Also this eliminate possibility to differ WINRT and WINDOWS_MOBILE if required in future updates.

Current GTest version supports WINRT and WP now ([one of related revisions](https://code.google.com/p/googletest/source/detail?r=690)). There is also some work in progress [there](https://github.com/MSOpenTech/opencv/tree/test_gtest_upd).

Could you please clarify reasons why OpenCV doesn't use GTest as external library to have possibility of continuous updates? And are there any reasons that GTest-related files were not updated for a long time?